docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,6 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
```

Comment:
Suggestion: Describe the method here.

For example: By default, binding metadata is exposed. (if it is default, then this method must come as the first option in the bullet list)

Question: Is method 2 automated by the SB operator?